.github/workflows/publish-to-npm.yml

```diff
@@ -15,10 +15,10 @@ jobs:
   verify-candidate-tag:
     runs-on: ubuntu-latest
     steps:
-      # Check out the release branch, and get its head commit as output for later.
+      # Check out the main-5 branch, and get its head commit as output for later.
```

Comment:
Remind me... what triggers publish_to_npm?

Reply:
The creation of a github release. But you're making a good point because I believe the version that will be triggered is the one on `dev`, which is a problem for us. I'll do more research into this.